test/spec/idx/idxState/unit/v1/generateIdxAction.test.js

```diff
@@ -11,50 +11,60 @@
  */
 
 
-import generateIdxAction from '../../../../../../lib/idx/idx-js/v1/generateIdxAction';
+import generateIdxAction from '../../../../../../lib/idx/idxState/v1/generateIdxAction';
 
 const mockIdxResponse = require('../../mocks/request-identifier');
 const mockPollingIdxResponse = require('../../mocks/poll-for-password');
 
-const { Response } = jest.requireActual('cross-fetch');
+jest.mock('../../../../../../lib/http', () => {
+  const actual = jest.requireActual('../../../../../../lib/http');
+  return {
+    ...actual,
+    httpRequest: () => {}
+  };
+});
+
+const mocked = {
+  http: require('../../../../../../lib/http'),
+};
 
 const deepClone = ( target ) => JSON.parse(JSON.stringify( target ));
-const mockResponse = ( respondWith ) => Promise.resolve( new Response( JSON.stringify( respondWith) ) );
+const mockResponse = ( respondWith ) => Promise.resolve(respondWith);
 
-// import targets for mockery
-import fetch from 'cross-fetch';
-import { makeIdxState } from '../../../../../../lib/idx/idx-js/v1/makeIdxState';
+import { makeIdxState } from '../../../../../../lib/idx/idxState/v1/makeIdxState';
+import { AuthApiError } from '../../../../../../lib/errors';
 
-jest.mock('cross-fetch');
 /*
   Doing a jest.mock('../../src/makeIdxState') has problems with jest.mock causing the test to hang
   and spikes up the CPU usage for the current node process.
   Alternative mocking approach: https://jestjs.io/docs/en/es6-class-mocks
 */
 const mockMakeIdxState = jest.fn();
-jest.mock('../../../../../../lib/idx/idx-js/v1/makeIdxState', () => ({
+jest.mock('../../../../../../lib/idx/idxState/v1/makeIdxState', () => ({
```

Comment:
maybe in a future refactor we can map `../../../../../../lib/` to an alias in jest config with `moduleNameMapper `